apps/admin-gui/src/app/shared/components/dialogs/add-role-dialog/add-role-dialog.component.ts

```diff
@@ -36,10 +38,12 @@ export class AddRoleDialogComponent implements OnInit {
   @Output() submitForm = new EventEmitter<AddRoleForm>();
 
   selectedRule: RoleManagementRules;
-  selectedEntity: PerunBean = null;
+  selected = new SelectionModel<PerunBean>(true, []);
```

Comment:
Since the selection does not clear itself on role change, if user chooses for example some groups under group admin, then swaps to resource admin instead and tries to set that role, an error is thrown